src/randomMindustry/mappers/block/blocks/RandomItemBridge.java

```diff
@@ -0,0 +1,33 @@
+package randomMindustry.mappers.block.blocks;
+
+import arc.math.Mathf;
+import mindustry.type.Category;
+import mindustry.world.blocks.distribution.ItemBridge;
+import randomMindustry.RMVars;
+import randomMindustry.mappers.item.ItemMapper;
+
+import static randomMindustry.mappers.block.BlockMapper.r;
+
+public class RandomItemBridge extends ItemBridge implements RandomBlock {
+    public static int lastTier = 1;
+    int tier = lastTier++;
+
+    public RandomItemBridge(String name) {
+        super(name);
+        size = 1;
+        health = Mathf.round(r.random(1, 5) * tier, 5);
+        fadeIn = moveArrows = false;
+
+        requirements(Category.production, ItemMapper.getItemStacks(tier - 1, 2, () -> r.random(6, 10)));
+        stats.add(RMVars.seedStat, RMVars.seedStatValue);
+
+        range = 0.03f * (tier * ItemMapper.maxTier / 12f);
```

Comment:
![image](https://user-images.githubusercontent.com/83781960/195153228-6c4d8987-f936-42d8-babb-9bcb35dc23a4.png)
cast to int NOW